modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/chart/bar/SimpleBarChart.es.js

```diff
@@ -40,6 +40,23 @@ export default ({data, height, totalEntries, width}) => {
 		setActiveIndex(index);
 	};
 
+	const CustomizedYAxisTick = ({payload, x, y}) => {
+		return (
+			<g transform={`translate(${x},${y})`}>
+				<text
+					className={`${
+						activeIndex !== null && activeIndex != payload.index
```

Comment:
@alinedoleron, is there any special reason for you use `!==` and `!=` on similar checks? If not, please make it more consistent. 😉

Reply:
Lack of attention. So sorry!